workers/unity/Assets/Gdk/TestUtils/HybridGdkSystemTestBase.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using NUnit.Framework;
+using Unity.Entities;
+
+namespace Improbable.Gdk.TestUtils
+{
+    public abstract class HybridGdkSystemTestBase
+    {
+        private static readonly Type GameObjectArrayInjectionHookType =
+            typeof(GameObjectEntity).Assembly.GetType("Unity.Entities.GameObjectArrayInjectionHook");
+
+        private static readonly Type TransformAccessArrayInjectionHookType =
+            typeof(GameObjectEntity).Assembly.GetType("Unity.Entities.TransformAccessArrayInjectionHook");
+
+        private static readonly Type ComponentArrayInjectionHookType =
+            typeof(GameObjectEntity).Assembly.GetType("Unity.Entities.ComponentArrayInjectionHook");
+
+        private InjectionHook gameobjectArrayInjectionHook;
+        private InjectionHook transformAccessArrayInjectionHook;
+        private InjectionHook componentArrayInjectionHook;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            gameobjectArrayInjectionHook = (InjectionHook) Activator.CreateInstance(GameObjectArrayInjectionHookType);
```

Comment:
Why do we need to do these reflection shenanigans?

Reply:
It's what we do in our bootstrap, I can copy the comment from there:

https://github.com/improbable/unity-gdk/blob/master/workers/unity/Assets/Playground/Scripts/Bootstrap.cs#L108-L125